packages/yoga/src/Checkbox/web/Checkbox.jsx

```diff
@@ -298,6 +301,8 @@ const Checkbox = ({
             {...restWithoutEvents}
             onChange={onChange}
             onClick={onClick}
+            aria-labelledby={ariaLabel ? undefined : checkboxLabelId}
+            aria-label={ariaLabel || undefined}
```

Comment:
The default value of `ariaLabel` is already `undefined`

```suggestion
            aria-label={ariaLabel}
```